plugins/woocommerce/src/Checkout/Enums/OrderStatus.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Checkout\Enums;
```

Comment:
Is `Automattic\WooCommerce\Checkout\Enums` perhaps too granular (how many more enum-style classes might we add under `Checkout`)? What about `Automattic\WooCommerce\Enums`, or perhaps the existing `Utilities` namespace?

(I don't have a super-strong feeling about this but, though it was worth some discussion before we cement it into the public API.)

Reply:
My idea here was to indeed introduce other enums later, but yeah, let's way for the need. I changed it to `Automattic\WooCommerce\Enums` as suggested 👍